examples/physics_ammo_instancing.html

```diff
@@ -137,26 +137,28 @@
 				controls.update();
 
 				animate();
+				
+				setInterval( () => {
 
-			}
+					let index = Math.floor( Math.random() * boxes.count );
 
-			function animate() {
+					position.set( 0, Math.random() + 1, 0 );
+					physics.setMeshPosition( boxes, position, index );
 
-				requestAnimationFrame( animate );
+					//
 
-				//
+					index = Math.floor( Math.random() * spheres.count );
 
-				let index = Math.floor( Math.random() * boxes.count );
+					position.set( 0, Math.random() + 1, 0 );
+					physics.setMeshPosition( spheres, position, index );
 
-				position.set( 0, Math.random() + 1, 0 );
-				physics.setMeshPosition( boxes, position, index );
+				}, 17 );
```

Comment:
Hmm... Why 17 milliseconds?

Reply:
I guess the OP wants to ensure that the related code block is executed 60 times per second (in order to imitate 60FPS).

Reply:
Yeah... I was just curious why `17` and not `16.66` or just `1000 / 60`.

Reply:
I'll merge and tweak.